tools/challenge-helper-scripts/create-project.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
```suggestion
```
I'm surprised that it's complaining about this, since it's part of node.  Do you have node 12 installed locally, by any chance?

Reply:
I did, but the warning has disappeared after I updated.